src/models/copy.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+// Model definition for a copy of a book.
+module.exports = (sequelize, DataTypes) => {
+  const Copy = sequelize.define('Copy', {
+    BookId: {
+      type: DataTypes.INTEGER,
+    },
+    copyAcquisitionDate: {
+      type: DataTypes.DATEONLY,
+      allowNull: false,
+      validate: {
+        notEmpty: true
+      }
+    },
+    copyNumber: {
```

Comment:
What's this for?

Reply:
How does a real-world person distinguish _this_ physical book from _that_ physical book so we know which one you've got on loan? The `copyNumber` written on the physical book does that, distinguishing between the different copies of the books. 

But perhaps a comment in the code explaining that would be good.